academy/golang-intro/1_install.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Go Introduction"
+order: 
+description: Get started with Golang
+tag: deep-dive
+---
+
+# Go Introduction
+
+Golang (Go) is an open-source, statically typed, and compiled programming language introduced by Google and first released in 2012. It has a BSD-style license. Its Git repository is located at [https://go.googlesource.com/go](https://go.googlesource.com/go).
+
+Start with the basics. In this section, you will install Go and learn some general things about it.
+
+
+## Install
+
+Installing Go depends on your machine. See the [Getting Started Page](https://golang.org/doc/install) and follow the instructions to install Go.
+
+<HighlightBox type="tip">
+
+[Gimme](https://github.com/travis-ci/gimme) is a useful script for Go.
+
+</HighlightBox>
+
+## Workflow
+
+Go has a different workflow from most other programming tools. You work with a single workspace and have all Go code contained in this workspace. The `GOPATH` environment variable specifies the location of your workspace. You can print it by typing `$ go env GOPATH` in your console. For more information about `GOPATH` type `$ go help gopath`.
+
+A version control system is helpful because you have only one workspace. You should use Git for this.
+
+
+## Hello, World!
+
+Let's start with the ever-popular "Hello, World!" program. Let's see how it looks in Go:
+
+```golang
+package main
+import "fmt"
+func main() {
+    fmt.Printf("Hello, World!\n")
+}
+```
+
+<HighlightBox type="tip">
+
+[Test it online](https://go.dev/play/p/1u5bSZlh80h).
+
+</HighlightBox>
+
+<HighlightBox type="info">
+
+Function names determine visibility outside of packages. `Printf`'s first character is upper case and this means it's visible outside of `fmt`. Package names are always written in lower case, like `fmt` (short for "format").
+
+</HighlightBox>
+
+Every Go program starts with the package declaration. Each package consists of one or more Go source files in a single directory. If the name of the package is `main`, Go creates an executable file.
+
+After package declaration, import the package `fmt`. Note that the package name is the last element of the package's file path. For example, if you import the package "lib/math", then you use it as "math".
+
+`fmt` implements input and output, and is part of Go's standard library. Later you are going to describe some of its functions in more detail.
+
+Execution starts in the function called `main()`.
+
+This function simply calls the Go I/O function `Printf()` from the package `fmt`.
+
+Now compile this program. Create the file `hello.go` in the `$GOPATH/src/hello` directory. In a terminal:
```

Comment:
This is not necessary anymore to have the file here. We should just tell them to create a folder somewhere of their choice.

Reply:
Tackled in the last commit with minor edit.